package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Online University Staff Directory with ChatBot",
   "main": "server.js",
   "scripts": {
-    "start": "NODE_ENV=development node server.js",
+    "start": "NODE_ENV=development node backend/server.js",
```

Comment:
should make it easier to start up the server from the root of the project. If you want to further simplify your development you could use: https://nodemon.io/